From e011f4c4487de6db17dbbc7a048db7f473c9d6fe Mon Sep 17 00:00:00 2001
From: Vabhav Sharma <vabhav.sharma@nxp.com>
Date: Tue, 26 Mar 2019 19:40:31 +0530
Subject: [PATCH 676/706] dt-bindings: serial: fsl-lpuart: add endianness
 property

add boolean property "little-endian" ti lpuart nodes compatible
with fsl,ls1021a-lpuart" should contain only when IP is LE, not
required when IP is big-endian

Signed-off-by: Vabhav Sharma <vabhav.sharma@nxp.com>
(cherry picked from commit b3a51aa8e6a3f6fb6452dd429f658e4b88320ff5)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 Documentation/devicetree/bindings/serial/fsl-lpuart.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/devicetree/bindings/serial/fsl-lpuart.txt b/Documentation/devicetree/bindings/serial/fsl-lpuart.txt
index 6bd3f2e93d61..202e6b43b021 100644
--- a/Documentation/devicetree/bindings/serial/fsl-lpuart.txt
+++ b/Documentation/devicetree/bindings/serial/fsl-lpuart.txt
@@ -12,6 +12,9 @@ Required properties:
 - interrupts : Should contain uart interrupt
 - clocks : phandle + clock specifier pairs, one for each entry in clock-names
 - clock-names : should contain: "ipg" - the uart clock
+- little-endian: For lpuart compatible with fsl,ls1021a-lpuart" should contain
+                 the boolean property "little-endian" only when IP is LE, not
+                 required when IP is big-endian
 
 Optional properties:
 - dmas: A list of two dma specifiers, one for each entry in dma-names.
-- 
2.17.1

